integration-tests/src/test/java/io/druid/tests/indexer/ITKafkaTest.java

```diff
@@ -0,0 +1,286 @@
+package io.druid.tests.indexer;
+
+/*
+This is a test for the kafka firehose.
+*/
```

Comment:
i believe this is a class comment, conventionally we keep it close to class declaration statement, can you move it to line-40 before class declaration.